tests/custom_mount_test.go

```diff
@@ -0,0 +1,56 @@
+package mos_test
+
+import (
+	"context"
+	"fmt"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	. "github.com/spectrocloud/peg/matcher"
+)
+
+var _ = Describe("kairos test custom user mounts", Label("custom-mounts-test"), func() {
```

Comment:
this adds a new test suite, however, as-is won't be called by our runners, care to add it?

Reply:
done.